### ISSUE TITLE
Support normalize false in metrics/classification/zeroOneLoss

### ISSUE THREAD

**Issue by JasonShin** [machinelearnjs/machinelearnjs#21]
* **I'm submitting a ...**
[/] feature request

* **Summary**
`zeroOneLoss` Should also have an implementation for `normalize:false`

* **Reference**

Kalimdor source location: https://github.com/JasonShin/kalimdorjs/blob/master/src/lib/metrics/classification.ts#L112

Scikit-learn reference: http://scikit-learn.org/stable/modules/generated/sklearn.metrics.zero_one_loss.html

**Comment by JasonShin** [comment]
This is easy to implement now because we have `normalize` API
